labcontrol/gui/templates/plate.html

```diff
@@ -159,31 +159,46 @@ <h4>Plate map</h4>
 
     <div id='plate-map-div' class='spreadsheet-container'></div>
 
-    <!-- Plate map legend div -->
+    <!-- Plate map legend div
+        Note that well-unknown and well-prev-plated are mutually exclusive
+        while well-duplicated is not. A duplicated value can appear in the
```

Comment:
minor but the wording of this comment is a bit unclear, it's initially confusing as to what `well-duplicated` "is not".

something like "Although `well-prev-plated` is mutually exclusive with `well-unknown`, `well-duplicated` is not." would make more sense IMO